simple_auth/lib/src/oauth/oauthAccount.dart

```diff
@@ -8,9 +8,13 @@ class OAuthAccount extends Account {
       this.refreshToken,
       this.expiresIn,
       this.created,
-      this.scope,
+      List<String> scope,
       Map<String, String> userData = const {}})
-      : super(identifier, userData: userData) {}
+      : super(identifier, userData: userData) {
+    if (scope == null) {
```

Comment:
Why only set the scope when it is null? What about when it has values? This will break lots of other things. Scope is needed.

Reply:
Sorry for the late reply. The scope should be set only if it is NOT null.  
There's a crash while parse null value from Json string to Object. I have updated this PR.